### ISSUE TITLE
Evaluate usage of Google Recaptcha

### ISSUE THREAD

**Issue by Pascalmh** [webSPELL/webSPELL-4.2.3#113]
As it came up in the Comments on webSPELL.org - should we add the option to use https://www.google.com/recaptcha/ or a diffrent captcha? Should there be a Option to switch one or another solution on - is this not necessary because the spam api will take care of it?

**Comment by derchrisuk** [comment]
If, at all, this would be a new feature for a forthcoming 4.3.x.
But in general, I too, think the Captcha needs a revision.

**Comment by DDaems** [comment]
I personnaly dislike using captchas, certainly the ones that are timeconsuming.
And i think all enduser will agree and in the end its barely affective against bots these days.
(but spam is even worse for the end-user)

Googles new approach, is allot userfriendly.

Also a possible solution is, removing it. And use some hidden fields with generated keys.
Like David Walsh talks about here:
http://davidwalsh.name/wordpress-comment-spam

**Comment by Phhere** [comment]
I do not think that this is a good solution because it is much easier to break than a captcha
the bot software just has to learn that there is a parameter hidden in the javascript

**Comment by DDaems** [comment]
Yes after reading this case-study of headless browser usage
http://blog.vamsoft.com/2014/07/09/headless-browser-use-in-web-forum-spam/
It doesn't seem to be a good approach anymore.

There a still ways to detect it, but it doesn't seem suiting for a project like this.
Maybe something more intresting for a addon/plugin writer.

Google recaptacha is probably the best solution.
But a option to disable it, incase someone prefers a addon with other techniques.
recieves a +1 from me. (unless this is allready built in, didn't check that really)

**Comment by Pascalmh** [comment]
We will evaluate the usage and settings of our current captcha - as this Ticket is already set to "post 4.3" we will revisit this Issue after the release.

**Comment by Getschonnik** [comment]
We have the Google reCaptcha integrated. In Admin Panel, the key must be deposited. There it is also possible to de-activate the reCaptcha. It is then exchanged between the webSPELL-Captcha and reCaptcha.

Implementation for the registration and comments already done. Guestbook soon..

The first tests are successful.
